tests/ui/resolve/resolve-conflict-extern-crate-vs-extern-crate.stderr

```diff
@@ -1,8 +1,11 @@
 error[E0259]: the name `std` is defined multiple times
+  --> $DIR/resolve-conflict-extern-crate-vs-extern-crate.rs:2:1
+   |
+LL | extern crate std;
+   | ^^^^^^^^^^^^^^^^^ `std` reimported here
```

Comment:
I'm very confused; why did the diagnostic emitter deem this span available now but not before? You only offset the crate declaration... seems fishy unless I'm missing something.